src/rest/Facet/FacetSortCriteria.ts

```diff
@@ -1,7 +1,23 @@
+/**
+ * The allowed [`sortCriteria`]{@link MLFacet.options.sortCriteria} option
+ * values for the `MLFacet` component.
+ */
 export enum FacetSortCriteria {
+  /**
+   * Sort facet values in descending score order.
+   *
+   * Facet value scores are based on number of occurrences and position in the
+   * ranked query result set.
+   *
+   * **Note:** The Coveo ML _Facet Sense_ feature only works with the `score`
+   * sort criterion.
+   */
   score = 'score',
-  alphaascending = 'alphaascending',
-  alphadescending = 'alphadescending',
+
+  /**
+   * Sort facet values in ascending alphanumeric order.
+   */
+  alphanumeric = 'alphanumeric',
```

Comment:
`score` and `alphanumeric` are the only available values in the Search API.

As discussed with @ThibodeauJF, `occurrences` may be supported eventually, which is why I left it here (and did not document it).